source/features/sync-pr-commit-title.tsx

```diff
@@ -79,28 +79,18 @@ async function updateCommitTitle(event: Event): Promise<void> {
 }
 
 function disableSubmission(): void {
-	deinit();
```

Comment:
Doesn't look right? We intentionally drop every listener because we "cancel" the feature entirely. 

If there's no simple solution to this, please revert this file.

Reply:
> We intentionally drop every listener because we "cancel" the feature entirely.

Couldn't the same effect be achieved by adding `{once: true}` to every delegated listener?

Reply:
No because they all need to be removed at once, they should not be listened at all after the feature is disabled